editor/sidebar/post-taxonomies/index.js

```diff
@@ -35,9 +35,11 @@ class PostTaxonomies extends Component {
 
 	componentDidMount() {
 		this.fetchTaxonomies = new wp.api.collections.Taxonomies()
-			.fetch()
```

Comment:
Maybe not here, but we should see about upgrading this component to using the `withAPIData` higher-order component.

https://github.com/WordPress/gutenberg/tree/master/components/higher-order/with-api-data

Reply:
already done and merged, needs rebase :)

Reply:
I will rebase and use the component.